static/gsApp/views/amCheckout/utils.tsx

```diff
@@ -586,12 +586,13 @@ export async function submitCheckout(
   api: Client,
   onFetchPreviewData: () => void,
   onHandleCardAction: (intentDetails: IntentDetails) => void,
+  expand: string[],
   onSubmitting?: (b: boolean) => void,
   intentId?: string,
   referrer = 'billing',
   shouldUpdateOnDemand = true
 ) {
-  const endpoint = `/customers/${organization.slug}/subscription/`;
+  const endpoint = `/customers/${organization.slug}/subscription/${expand.length > 0 ? `?expand=${expand.join(',')}` : ''}`;
```

Comment:
this is necessary to ensure the response includes the invoice for the later confirmation screen